### ISSUE TITLE
[ISSUE] when I drag .psb into FreeMoteViewer.exe, nothing happen

### ISSUE THREAD

**Issue by killeromg007** [UlyssesWu/FreeMote#136]
* Read [wiki](https://github.com/UlyssesWu/FreeMote/wiki)
* Provide game name and platform (PC/Android/PSP etc.)
* Provide PSB file samples
* Program output, or screenshots

**Comment by UlyssesWu** [comment]
I'll leave you some time to fill the required info. Otherwise just closing.

**Comment by killeromg007** [comment]
ok

**Comment by killeromg007** [comment]
https://drive.google.com/file/d/19p6EUCj4ulX0cpD3PhL_9Qca7P4gbu8X/view?usp=sharing

**Comment by killeromg007** [comment]
pls

**Comment by killeromg007** [comment]
@UlyssesWu

**Comment by UlyssesWu** [comment]
Try: https://github.com/UlyssesWu/FreeMote/actions/runs/11216171804

**Comment by killeromg007** [comment]
> Try: https://github.com/UlyssesWu/FreeMote/actions/runs/11216171804

what

**Comment by killeromg007** [comment]
Duplicate of #
the link ur gave me is not working like past as well

**Comment by killeromg007** [comment]
@UlyssesWu

**Comment by UlyssesWu** [comment]
1. Make sure you're using the exe with my latest link.
2. go to `lib` folder.
3. Check if `emotedriver.dll`, `FreeMote.NET.dll` are blocked: https://analyticalgraphics.my.salesforce-sites.com/faqs/articles/Knowledge/How-to-unblock-dll-files-downloaded-from-the-internet?retURL=%2Ffaqs%2Fapex%2Ffaq%3FTopics%3DEducation&popup=true
4. It Works On My Computer™, so just help yourself.
5. Learn to be polite, kid. Stop comment just a few words for many times, or immediately @ people to bothering them with notification email. I'm not your free customer service. You will receive no more help from me. Good luck.
